src/grafana_dashboards/MongoDB_ReplSet_Summary.json

```diff
@@ -1227,19 +1229,18 @@
             },
             "gridPos": {
                 "h": 8,
-                "w": 12,
-                "x": 12,
-                "y": 40
+                "w": 8,
+                "x": 0,
```

Comment:
same here

Reply:
Might not be needed indeed.
I will spin everything up again so I can check